papers/ring_vrf/abstract-short.tex

```diff
@@ -1,35 +1,35 @@
 
 \def\eprintsmallskip{\smallskip}{}%
 We introduce a new cryptographic primitive, aptly named
- \emph{ring verifiable random functions (ring VRF)},
- as a broadly superior approach to anonymous credentials.
+\emph{ring verifiable random functions (ring VRF)}.
 % Anonymized
 %\eprint{Ring VRFs are}{We introduce ring VRFs, which are}
-Ring VRFs are (anonymized) ring signatures that prove correct evaluation
-of some authorized signer's PRF, while hiding the specific signer's
-identity within some set of possible signers, known as the ring. We design a ring VRF protocol which has efficient instantiations with our novel {\em zero-knowledge continuation} technique.
+Ring VRFs are ring signatures that also prove correct evaluation
+of a pseudo-random function (PRF) by a signer, while hiding the  signer's
+identity within some set of signers, known as the ring. 
```

Comment:
This looses something I think.

Reply:
It sucks either way. If you say that it's a "ring signature" you assume that the reader is familiar. Then "while hiding the  signer's identity within some set of signers, known as the ring" is excessive. Otherwise we should break the sentence in 2: "ring signature is... ring VRF is a ring signature that..."